hack/verify-generated-crd.sh

```diff
@@ -10,22 +10,27 @@ function verify_crd {
   fi
 }
 
-verify_crd \
+
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" ]] && verify_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" \
   "manifests/00-custom-resource-definition.yaml"
 
-verify_crd \
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-CustomNoUpgrade.crd.yaml" ]] && verify_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-CustomNoUpgrade.crd.yaml" \
   "manifests/00-custom-resource-definition-CustomNoUpgrade.yaml"
```

Comment:
Again, what do you think about changing `verify_crd` instead, as follows?
```bash
function verify_crd {
  local SRC="$1"
  local DST="$2"
  if [[ -e "$SRC" ]]
  then
    if [[ -e "$DST" ]]
    then
      if ! diff -Naup "$SRC" "$DST"; then
        echo "inconsistent CRD: $SRC => $DST"
      fi
    else
      echo "missing CRD: $SRC => $DST"
      exit 1
    fi
  else
    if [[ -e "$DST" ]]
    then
      echo "extra CRD: $DST"
      exit 1
    fi
  fi
}
```

Reply:
yes looks good thanks!

Reply:
done